sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java

```diff
@@ -1898,7 +1898,8 @@ public void testBigQueryQuerySourceInitSplit() throws Exception {
         true /* useLegacySql */,
         fakeBqServices,
         TableRowJsonCoder.of(),
-        BigQueryIO.TableRowParser.INSTANCE);
+        BigQueryIO.TableRowParser.INSTANCE,
+        "BATCH");
```

Comment:
Please add a test that sets priority in BigQueryIO to BATCH/STREAMING and confirms that request to BQ contains the correct priority.